include/knightos/display.h

```diff
@@ -97,8 +97,20 @@ void draw_short(SCREEN *screen, unsigned char x, unsigned char y, unsigned short
 **/
 void draw_byte(SCREEN *screen, unsigned char x, unsigned char y, unsigned char value);
 
-/*.*
+/**
+ * Draws a signed short at x, y
```

Comment:
No real reason to have these be three lines when they can be one; all it does is waste space

Reply:
Meh, style is unimportant.